runtime/parachains/src/paras_inherent.rs

```diff
@@ -793,39 +792,34 @@ pub(crate) fn sanitize_bitfields<T: crate::inclusion::Config, const EARLY_RETURN
 		// exposes a DoS vector where a block author introduces bad bitfields to block processing
 		// of remaining bitfields.
 		if unchecked_bitfield.unchecked_payload().0.len() != expected_bits {
-			println!("B");
 			continue
 		}
 
 		if unchecked_bitfield.unchecked_payload().0.clone() & disputed_bitfield.0.clone() !=
 			all_zeros
 		{
-			println!("C");
 			continue
 		}
 
 		if !last_index.map_or(true, |last| last < unchecked_bitfield.unchecked_validator_index()) {
-			println!("D");
 			continue
 		}
 
 		if unchecked_bitfield.unchecked_validator_index().0 as usize >= validators.len() {
-			println!("E");
 			continue
 		}
 
 		let validator_index = unchecked_bitfield.unchecked_validator_index();
 
 		let validator_public = &validators[validator_index.0 as usize];
 
-		// only check the signatures when returning early
+		// Only check the signatures when returning early
 		if EARLY_RETURN {
```

Comment:
Renaming this is probably a good idea :)

Reply:
7d330e8